tests/test_oauth.py

```diff
@@ -24,7 +28,7 @@ def test_get_oauth_redirect_url_when_not_set(self):
         redirect_url = csm.get_oauth_redirect_url(provider="onadata")
         assert redirect_url is None
 
-    def test_get_oauth_redirect_url_when_set(self):
+    def test_get_oauth_redirect_url_when_set(self):  # pylint: disable=R0201
```

Comment:
It would be really nice to use the english-like name of the rule being ignored, if possible, instead of just the code.

Reply:
I tend to use codes when I start hitting line length limits, the names can be really long.